onfido/onfido.py

```diff
@@ -33,3 +45,23 @@ def __init__(self, api_token, region, timeout=None):
             pass
         elif "api.onfido.com" in region:
             raise OnfidoRegionError("The region must be one of Region.EU, Region.US or Region.CA. We previously defaulted to Region.EU, so if you previously didn’t set a region or used api.onfido.com, please set your region to Region.EU")
+
+class AsyncApi:
+    def __init__(self, api_token, region, aio_session, timeout=None):
+        self.applicant = AsyncApplicants(api_token, region, aio_session, timeout)
```

Comment:
In aio_resource the private class vars are declared using _ as "private" vars, why here it's not done the same way?

Reply:
same here, chamaleon
Or I don't understand what you mean ?

Reply:
nevermind, i missread here